mmv1/third_party/terraform/utils/provider_test.go.erb

```diff
@@ -682,6 +683,473 @@ func TestProvider_providerConfigure_project(t *testing.T) {
 	}
 }
 
+func TestProvider_providerConfigure_billingProject(t *testing.T) {
+
+	cases := map[string]struct {
+		ConfigValues     map[string]interface{}
+		EnvVariables     map[string]string
+		ExpectedValue    string
+		ExpectError      bool
+		ExpectFieldUnset bool
```

Comment:
Look into removing/replacing this

Reply:
On second thought, nah. Using this is more explicit. If there's an issue then this can be addressed in refactoring after this PR.